src/ontology/mondo-edit.obo

```diff
@@ -615820,6 +615823,29 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-7638-4659
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8866" xsd:anyURI
 
+[Term]
+id: MONDO:1060108
+name: PURA-related severe neonatal hypotonia-seizures-encephalopathy syndrome
+def: "A rare neurologic disease characterized by neonatal hypotonia, global developmental delay, feeding difficulties, and often seizures or seizure-like episodes. Other frequently observed signs and symptoms include variable dysmorphic features, myopathic facies, respiratory problems, and visual abnormalities, such as strabismus or esotropia. Brain imaging may show delayed myelination and other white matter abnormalities." [Orphanet:438213]
+subset: ordo_disorder {source="Orphanet:438213"}
+subset: otar {source="MONDO:OTAR"}
+synonym: "PURA syndrome" NARROW [PMID:28448108]
+synonym: "PURA-related neurodevelopmental disorders" EXACT [https://orcid.org/0000-0001-9310-0163, PMID:28448108]
+xref: ICD10CM:G40.4 {source="Orphanet:438213/attributed", source="Orphanet:438213/ntbt", source="Orphanet:438213"}
+xref: Orphanet:438213 {source="MONDO:equivalentTo"}
+is_a: MONDO:0003847 ! hereditary disease
+is_a: MONDO:0015159 {source="Orphanet:438213", source="https://orcid.org/0000-0001-9310-0163"} ! multiple congenital anomalies/dysmorphic syndrome-intellectual disability
+relationship: curated_content_resource https://www.malacards.org/card/pura_related_neurodevelopmental_disorders {source="MONDO:MalaCards"}
+relationship: has_characteristic HP:0000006 {source="Orphanet:438213"} ! Autosomal dominant inheritance
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9701 {source="Orphanet:438213"} ! PURA
```

Comment:
I don't think this is correct. 
The children will inherit this gene association. And we know that the "microdeletion version" of the disease is not caused by a variation/mutation in the gene, but by a chromosomal deletion.